solutions/Making-choices-choices-with-functions-R.txt

```diff
@@ -0,0 +1,3 @@
+[1] "CM - 0.866"
```

Comment:
I acknowledge that I have failed to devise an option where the choice returns a value that is NOT a string.